app/views/component_detail.html

```diff
@@ -51,7 +51,7 @@ <h1 class="component-header">{{module_name}}
 			<div data-o-grid-colspan="9" class="demos-wrap">
 		{% endif %}
 			{% for demo in demos %}
-				<div class="demo-container{% if demo.expanded %} visible{% endif %} with-source" id="demo-{{ demo.name|slugify}}">
+				<div class="demo-container{% if not demo.hidden %} visible{% endif %} with-source" id="demo-{{ demo.name|slugify}}">
```

Comment:
The `visible` class is going to be removed in the new design which this will be for so we can probably just use the `{% if not demo.hidden %}` to wrap the full demo markup.